include/deal.II/lac/petsc_matrix_base.h

```diff
@@ -784,6 +784,43 @@ namespace PETScWrappers
     void Tvmult_add (VectorBase       &dst,
                      const VectorBase &src) const;
 
+    /**
+     * Perform the matrix-matrix multiplication <tt>C = A * B</tt>, or, if an
```

Comment:
Here and below, there is no need for the asterisk to indicate multiplication. Just write this as `$C = AB$`, `$C = A \text{diag}(V) B$`, etc.